hello_network/rel/config.exs

```diff
@@ -0,0 +1,39 @@
+use Mix.Releases.Config,
+    # This sets the default release built by `mix release`
+    default_release: :default,
+    # This sets the default environment used by `mix release`
+    default_environment: :dev
+
+# For a full list of config options for both releases
+# and environments, visit https://hexdocs.pm/distillery/configuration.html
+
+
+# You may define one or more environments in this file,
+# an environment's settings will override those of a release
+# when building in that environment, this combination of release
+# and environment configuration is called a profile
+
+environment :dev do
+  set cookie: :"Lm>b)%ecJf:|am;Vue/a[0.5Bgf1}c6yQpwbbQg*EyJTmz>S{!CoU4GeiTZwKwGd"
```

Comment:
Hmm. This is the auto-generated file that `mix nerves.release.init` created, and I don't believe it actually gets used to set the cookie in Nerves, right @mobileoverlord?

I think you would set the cookie by putting `-cookie <something>` in `rel/vm.args` instead of having it come from this file. I believe Distillery uses this configuration to create a release, but I'm not sure how `rel/config.exs` and `rel/vm.args` interact.